code/src/index.css

```diff
@@ -1,13 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,400;0,500;0,600;0,700;0,800;1,400&family=Nunito:wght@200;400;600;700;900&family=Plus+Jakarta+Sans:wght@200;300;400;500;700;800&display=swap');
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont,  "Plus Jakarta Sans", "Roboto", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  background: url('../src/assets/orange.jpg') no-repeat center center fixed;
```

Comment:
Love this background image! My son just said: It was very much more nice then your background (just one color)